tests/conftest.py

```diff
@@ -73,20 +89,24 @@ def __init__(self, ctn_name, pid, i):
         if os.path.exists("/var/run/netns/%s" % self.nsname):
             self.cleanup = False
         else:
-            os.system("ip netns add %s" % self.nsname)
+            ensure_system("ip netns add %s" % self.nsname)
 
             # create vpeer link
-            os.system("ip link add %s type veth peer name %s" % (self.nsname[0:12], self.vifname))
-            os.system("ip link set %s netns %s" % (self.nsname[0:12], self.nsname))
-            os.system("ip link set %s netns %d" % (self.vifname, pid))
+            ensure_system("ip link add %s type veth peer name %s" % (self.nsname[0:12], self.vifname))
+            ensure_system("ip link set %s netns %s" % (self.nsname[0:12], self.nsname))
+            ensure_system("ip link set %s netns %d" % (self.vifname, pid))
 
             # bring up link in the virtual server
-            os.system("ip netns exec %s ip link set dev %s name eth0" % (self.nsname, self.nsname[0:12]))
-            os.system("ip netns exec %s ip link set dev eth0 up" % (self.nsname))
-            os.system("ip netns exec %s ethtool -K eth0 tx off" % (self.nsname))
+            ensure_system("ip netns exec %s ip link set dev %s name eth0" % (self.nsname, self.nsname[0:12]))
+            ensure_system("ip netns exec %s ip link set dev eth0 up" % (self.nsname))
+            ensure_system("ip netns exec %s ethtool -K eth0 tx off" % (self.nsname))
 
             # bring up link in the virtual switch
-            os.system("nsenter -t %d -n ip link set dev %s up" % (pid, self.vifname))
+            ensure_system("nsenter -t %d -n ip link set dev %s up" % (pid, self.vifname))
+
+        # disable arp, so no neigh on vEthernet(s)
+        # Note: outside the if-else, so existing VS container could be fixed
+        ensure_system("nsenter -t %d -n ip link set arp off dev %s" % (pid, self.vifname))
```

Comment:
I think arp is needed for neighborsyncd and fdb function test?  The vEthernet interfaces here are actually mapped to Ethernet interfaces in vs (thought I'm not clear how that is done with the docker --network container:sw magic)

Reply:
I disabled arp on only vEthernetXX. ARP and neighbors on EthernetXX are not affected during my experiment.